imagery/i.ortho.photo/lib/find_init.c

```diff
@@ -8,11 +8,16 @@
 int I_find_initial(char *group)
 {
     char *element;
+    int file_exists;
 
     element = (char *)G_malloc(80 * sizeof(char));
 
-    if (group == NULL || *group == 0)
+    if (group == NULL || *group == 0) {
+        G_free(element);
```

Comment:
you could return 0 before allocating element

Reply:
done changes